index.html

```diff
@@ -312,6 +312,36 @@ <h4>id</h4>
         </pre>
       </section>
 
+      <section>
+        <h4>alsoKnownAs</h4>
+
+        <table class="simple" style="width: 100%;">
+          <thead>
+          <tr>
+            <th>Normative Definition</th>
+            <th>JSON-LD</th>
+          </tr>
+          </thead>
+          <tbody>
+          <tr>
+            <td>
+              <a href="https://www.w3.org/TR/did-core/#also-known-as">DID Core</a>
+            </td>
+            <td>
+              <a href="https://www.w3.org/ns/did/v1">DID Core</a>
```

Comment:
Actually I'm not 100% if this is correct, or if we should list `https://www.w3.org/ns/activitystreams` instead. @rhiaro could you advise?

Reply:
This is correct... and the vocabulary URL for `alsoKnownAs` is https://www.w3.org/ns/activitystreams#alsoKnownAs as defined in https://www.w3.org/ns/did/v1